modules/ROOT/pages/update-streams.adoc

```diff
@@ -54,9 +54,11 @@ Switching between streams may introduce regressions or bugs due to skipping upda
 sudo systemctl stop zincati.service
 
 # Perform the rebase to a different stream
+# Supported architectures: aarch64, ppc64le, x86_64
```

Comment:
We don't have ppc64le yet, right?

Reply:
Oops, got confused looking at the different refs on the remote